server/sched.go

```diff
@@ -592,6 +617,13 @@ func (runner *runnerRef) needsReload(ctx context.Context, req *LlmRequest) bool
 	// Normalize the NumCtx for parallelism
 	optsExisting.NumCtx = optsExisting.NumCtx / runner.numParallel
 
+	// Compare cache types
```

Comment:
Similar to above, let's make one environment variable that sets both

Reply:
llama.cpp exposes both of these separately and I've seen people using different values for K and V. I'd opt for the pass through and flexibility at the small increase of complexity.

Reply:
Yes I agree it's worth having sperate options for them as there's slightly different trade offs and depending on your use case you might want to use the same for both or just one of them.